libcontainer/cgroups/fs/fs.go

```diff
@@ -241,6 +241,15 @@ func (m *manager) Path(subsys string) string {
 	return m.paths[subsys]
 }
 
+func (m *manager) Kill() error {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	// Since its not a subsystem and we dont want to regenerate path using pid
+	// strip dir path from any available subsystem, mostlikely cpu since its always there for v1
```

Comment:
1. `mostlikely` -> `most likely`

2. Period is missing at the end of the sentence.

3. I think (but not 100% sure) that we usually depend on `devices` controller. Check the code.

Reply:
Yeah this should be using `devices`.

Reply:
Fixed